extruct/_extruct.py

```diff
@@ -34,6 +35,7 @@ def extract(htmlstring, base_url=None, encoding="UTF-8",
                  '@type': 'example_type',
                  /* All other the properties in keys here */
                  }
+       return_html_node: if True, it includes HTML node of respective embedded metadata into the result.
```

Comment:
I think it makes sense to 

* and add a note about a key under which the node is returned;
* mention that node is lxml.etree.Element (or what type is the node?);
* wrap it to 79 columns;

Reply:
Sure, will do that :+1: